src/Main.java

```diff
@@ -0,0 +1,166 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Main obj = new Main();
+
+
+
+        System.out.println("Метод findSymbolOccurance");
+
+        obj.findSymbolOccurance("Java", 'a');
+
+        System.out.println();
+
+        System.out.println("Метод findWordPosition");
+
+
+        obj.findWordPosition("Apollo", "pollo" );
+        obj.findWordPosition("Aplle", "Plant" );
+
+        System.out.println();
+
+        System.out.println("Метод stringReverse");
+        obj.stringReverse("Hello");
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        obj.isPalindrome("ERE");
+        obj.isPalindrome("Allo");
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+
+
+
+
+
+
+
+
+    }
+
+    public void findSymbolOccurance (String str, char ch) {
+
+        char[] charArray = str.toCharArray();
```

Comment:
You should give a meaningful name to your char[], for example, 'symbols' or 'letters' instead of 'charArray'